docs/Sites/managed.md

```diff
@@ -82,7 +82,12 @@ Under the hosting section in Fleek, find the affected site, and visit the Settin
 
 ### Deleting a Site
 
-To delete a site, go to the site overview and click on the settings icon. In the general settings page, click on the **Delete Site** button. This will remove the site from your account and will no longer serving the content of your site.
+To delete a site, go to the site overview and click on the settings icon. In the general settings page, click on the **Delete Site** button. This will remove the site from your account and will no longer serving the content of your site. You will be asked to confirm the removal of all associated resources within the site. Ensure you check the box to the right to acknowledge this action before proceeding for:
```

Comment:
```suggestion
To delete a site, go to the site overview and click on the settings icon. In the general settings page, click on the **Delete Site** button. This will remove the site from your account and will no longer serve your site's content. You will be asked to confirm the removal of all associated resources within the site. Ensure you check the box to the right to acknowledge this action before proceeding for:
```